content/applications/general/users/portal.rst

```diff
@@ -63,3 +65,103 @@ user for that Odoo database.
    At any time, portal access can be revoked by navigating to the contact, clicking
    :menuselection:`Action --> Grant portal access`, and then unselecting the checkbox under the
    :guilabel:`In Portal` column and clicking :guilabel:`Apply`.
+
+.. _portal/login:
+
+Change portal username
+======================
+
+There may be times when a portal user wants to change their user login. This can be done by any user
+in the database with administrator access rights. The following process outlines the necessary steps
+to change the portal user login.
+
+.. seealso::
+   :doc:`See the documentation on setting access rights
+   </applications/general/users/access_rights>`.
+
+First, navigate to :menuselection:`Settings app --> Users`. Then, under :guilabel:`Filters`, select
+:guilabel:`Portal Users`, or select :guilabel:`Add Custom Filter` and set the following
+configuration :guilabel:`Groups` > :guilabel:`contains` > `portal`. After making this selection,
+search for (and open) the portal user that needs to be edited.
+
+Next, click :guilabel:`Edit` (if necessary), click into the :guilabel:`Email Address` field, and
+proceed to make any necessary changes to this field. The :guilabel:`Email Address` field is used to
+log into the Odoo portal.
+
+.. note::
+   Changing the :guilabel:`Email Address` (or login) only changes the *username* on the customer's
+   portal login.
+
+   In order to change the contact email, this change needs to take place on the contact template in
+   the *Contacts* app. Alternatively, the customer can change their email directly from the portal,
+   but the login **cannot** be changed. :ref:`See Change customer info <portal/custinfo>`.
```

Comment:
Remove capitalization– I think this was capitalized because its a ref to a title, I may be wrong about removing this though, so up to you!
```suggestion
   but the login **cannot** be changed. :ref:`See change customer info <portal/custinfo>`.
```